content/en/getting_started/synthetics/private_location.md

```diff
@@ -41,36 +45,41 @@ Once you've created and installed your private location, you can assign [Synthet
+   For more information, see [Private Locations Configuration Options][6] and [Run Synthetic Tests from Private Locations][7]. 
+7. Copy and paste your private location configuration file to your working directory.
+
+    **Note**: The configuration file contains secrets for private location authentication, test configuration decryption, and test result encryption. Datadog does not store the secrets, so store them locally before leaving the **Private Locations** creation form. **You need to be able to reference the secrets again in order to add more workers to your private location**. 
+8. When you are ready, click **View Installation Instructions**.
+9. Follow the installation instructions based on the environment you want to run the Private Location worker in.
+10. If you use Docker for example, launch your worker as a standalone container using the Docker `run` command and your configuration file:
 
     ```shell
     docker run --rm -v $PWD/worker-config-<LOCATION_ID>.json:/etc/datadog/synthetics-check-runner.json datadog/synthetics-private-location-worker
     ```
-
-8. If your private location reports correctly to Datadog, an `OK` health status displays on your **Private Locations** list in the **Settings**:
+11. If your private location reports correctly to Datadog, an `OK` health status displays under **Private Location Status** and on the **Private Locations** list in the **Settings** page:
 
     {{< img src="synthetics/private_locations/pl_health.png" alt="Private Location Health" style="width:100%;">}}
 
-    You should also receive private location logs on your terminal:
+    Additionally, you can see private location logs in your terminal:
 
     ```text
-    2019-12-17 13:05:03 [info]: Fetching 10 messages from queue - 10 slots available
-    2019-12-17 13:05:03 [info]: Fetching 10 messages from queue - 10 slots available
-    2019-12-17 13:05:04 [info]: Fetching 10 messages from queue - 10 slots available
+    2022-02-28 16:20:03 [info]: Fetching 10 messages from queue - 10 slots available
+    2022-02-28 16:20:03 [info]: Fetching 10 messages from queue - 10 slots available
+    2022-02-28 16:20:04 [info]: Fetching 10 messages from queue - 10 slots available
     ```
+12. Once you're done testing your internal endpoint, click **OK**.
+## Run Synthetic tests with your private location
 
-You can use your new private location just like a managed location to run Synthetic tests.
-
-## Run synthetic tests with your private location
+Use your new private location just like a managed location in your Synthetic tests.
 
-1. Create an [API test][2], [multistep API test][7], or [browser test][8] on any internal endpoint or application you want to monitor.
-2. Under **Private Locations**, select your new private location:
+1. Create an [API test][2], [multistep API test][8], or [browser test][9] on any internal endpoint or application you want to monitor.
+2. Under **Private Locations**, select your new private location(s):
```

Comment:
```suggestion
2. Under **Private Locations**, select your new private location:
```
`(s)` can be tricky for translators to translate. Do you mean plural or singular? I think in this GS, when you have just walked them through creating one, the singular is fine here. It's not confusing that the UI allows them to select multiple.